analyzers/packaging/SonarAnalyzer.CSharp.Styling.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>SonarAnalyzer.CSharp.Styling</id>
-    <version>9.24.0.0</version>
+    <version>9.25.0.0</version>
```

Comment:
Is the version bump script `set-version.ps1` already supporting the new analyzer? Or did you make these changes manually

Reply:
I only ran the script. So it is supported already.